### ISSUE TITLE
alt+l key binding breaks '@' symbol on German keyboards

### ISSUE THREAD

**Issue by jens-a-e** [ritwickdey/vscode-live-server#38]
This extension is the one I was looking for so long 😃 Very excellent and it works seamlessly. Thanks  a lot!

Unfortunately the default keybindings to start and stop the server break the `@` symbol on German keyboards - which is located on `alt-l`. I had to counter the bindings in my settings with:

```
[
    {
        "key": "alt+l o",
        "command": "-extension.liveServer.goOnline",
    },
    {
        "key": "alt+l c",
        "command": "-extension.liveServer.goOffline",
    }
]
```

As I am working mostly with Students new to the field this is rather cumbersome. Would there be a way to introduce the keybindings as optionally?

**Comment by jens-a-e** [comment]
Note: This is only concerns macOS users. The Windows and Linux kayboard layout is different.

**Comment by ritwickdey** [comment]
I don't have MacOS. Can you suggest me about the bindings for macOS?

**Comment by ritwickdey** [comment]
I can add platform specific key bindings!

**Comment by jens-a-e** [comment]
Excellent. That would be great! :+1:

On macOS I suggest it to be `cmd+L [O|C]`.

**Comment by tallon1252** [comment]
+1

**Comment by ritwickdey** [comment]
Hi, I'm sorry, I was too busy. Today (after few min) I'll fix the issue.. (I'm apologize for it)

**Comment by ritwickdey** [comment]
This issue has been resolved with v3.0.2.  Please update the extension and thanks for the reporting.

If you like the extension, [please drop a review to vscode marketplace](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer#review-details).

& let me know if it's still does not work for you. (I don't have MAC).

**Comment by jens-a-e** [comment]
Thanks for the fix!! :-)
Will try asap; also too busy at the moment.
